airflow/hooks/dbapi.py

```diff
@@ -128,6 +131,15 @@ def get_pandas_df(self, sql, parameters=None, **kwargs):
         with closing(self.get_conn()) as conn:
             return psql.read_sql(sql, con=conn, params=parameters, **kwargs)
 
+    def get_pandas_df_by_chunks(self, sql, parameters=None, **kwargs):
```

Comment:
Let’s make this

```suggestion
    def get_pandas_df_by_chunks(self, sql, parameters=None, *, chunksize, **kwargs):
```

So this function is directly usable as public API and a user won’t pass incorrect arguments in.

Reply:
Thank You for checking, I think it's good idea. I will make changes in PR tomorrow